src/client/store/metadata/hooks/useGetDashboard.ts

```diff
@@ -4,16 +4,12 @@ import { useAppDispatch } from 'client/store'
 import { useCountryRouteParams } from 'client/hooks/useRouteParams'
 
 import { MetadataActions } from '../slice'
-import { useDashboardItems } from './useDashboardItems'
 
 export const useGetDashboard = () => {
   const dispatch = useAppDispatch()
   const { assessmentName, cycleName, countryIso } = useCountryRouteParams()
-  const dashboardItems = useDashboardItems()
 
   useEffect(() => {
-    if (!dashboardItems) {
-      dispatch(MetadataActions.getDashboard({ assessmentName, cycleName, countryIso }))
-    }
-  }, [assessmentName, cycleName, countryIso, dispatch, dashboardItems])
+    dispatch(MetadataActions.getDashboard({ assessmentName, cycleName, countryIso }))
```

Comment:
Only change in PR:
Refetch items (eg. changing country -> region -> country)

Reply:
so i will introduce .region and .country ?

Reply:
https://github.com/openforis/fra-platform/pull/4027/commits/e60d9187d1eac1dab9faa46021a1eb5a607077c3

introduced here

this can later be expanded very easily to

[AreaCode]: dashboarditems

So we can think about each country / area having their own dashboard :tada: